pkg/consumer/consumer.go

```diff
@@ -35,16 +37,16 @@ type Interface interface {
 type consumer struct {
 	waitGr          *sync.WaitGroup
 	ctx             context.Context
-	conn            *pgx.ReplicationConn
-	dbCfg           pgx.ConnConfig
+	conn            *pgconn.PgConn
+	dbCfg           pgconn.Config
```

Comment:
Lots of changes like this where will now be referencing repos that were spun-out of the main `pgx` repo as of version 4.